src/tax-lot/tax-lot.repository.ts

```diff
@@ -129,4 +134,8 @@ export class TaxLotRepository {
       throw new DataRetrievalException();
     }
   }
+
+  async findTaxLotTilesets(params: { z: number; x: number; y: number }) {
+    return `${this.storageConfig.storageUrl}/tilesets/tax_lot/${params.z}/${params.x}/${params.y}.pbf`;
```

Comment:
Delegate the storageurl lookup and url tileset definition to the repository, under the argument that it is 'data'. The practical implication is that the "StorageConfig" disappears into the tax lot repo mock during testing